aim/web/ui/src/pages/RunDetail/RunOverviewTab/GitInfoCard/GitInfoCard.scss

```diff
@@ -0,0 +1,29 @@
+@use 'src/styles/abstracts' as *;
+
+.InfoSection {
+  overflow-y: scroll;
+  .InfoCard {
+    margin-right: toRem(40px);
+    min-width: fit-content;
+    &:last-child {
+      margin-right: 0;
+    }
+    .InfoCardLabel {
+      margin-bottom: toRem(4px);
+    }
+    .Icon__container {
+      color: $pico;
+    }
+    .InfoCardValue {
+      text-align: center;
+      background: $cuddle-10;
+      border: toRem(1px) solid $cuddle-50;
+      box-sizing: border-box;
+      border-radius: toRem(6px);
+      padding: toRem(6px) toRem(6px);
+      .InfoCardValueText {
+        margin-left: toRem(7px);
```

Comment:
let's set the margin-left correct value 

<img width="237" alt="qeewweq" src="https://user-images.githubusercontent.com/44981312/157090130-26c3eb9d-ab56-4784-9e1b-349781447091.png">